data/json/monstergroups/mammal.json

```diff
@@ -181,6 +189,16 @@
       { "monster": "mon_tusked_moose_calf", "weight": 50, "starts": "70 days", "conditions": [ "SPRING", "SUMMER" ] }
     ]
   },
+  {
+    "name": "GROUP_COYOTE",
+    "type": "monstergroup",
+    "//": "Not doing the whole weird thing with timers and weight halving for babies as the groups above me did. I have no idea what's that hoping to accomplish and nobody else seems to either.",
+    "monsters": [
+      { "monster": "mon_coyote", "weight": 38 },
+      { "monster": "mon_coyote_mutant_shark", "weight": 1, "starts": "272 days" },
```

Comment:
Is it correct? Aren't mutated animals should start born, like, from now, and just increase the rate the longer it goes? At least not three years after the cataclysm? And, from other side, unmutated kinds should stop born (ie using `ends`)

Reply:
> Is it correct? Aren't mutated animals should start born, like, from now, and just increase the rate the longer it goes?

You're right. The timer is assuming the mutant is born on the exact day the game starts. 272 days is how long it takes for a pup to grow to adulthood with default settings.

> And, from other side, unmutated kinds should stop born (ie using ends)

Not interested in that, really. Perhaps the odds could be increased down the line - but realistically they already will, because when mutant babies grow to adulthood they have 100% chance for their babies to be mutants. I don't think we need to juggle ``ends`` here.

Reply:
oh, i just checked the wrong mongroup